ngraph_bridge/ngraph_encapsulate_clusters.cc

```diff
@@ -793,6 +789,14 @@ Status EncapsulateClusters(
           }
           TF_RETURN_IF_ERROR(BackendManager::CreateBackend(
               op_backend_name));  // Created a backend here. must free it
+          // TranslateGraph must be called AFTER CreateBackend because some TF
+          // ops like CNMS and gather use backend specific nodes
+          TF_RETURN_IF_ERROR(Builder::TranslateGraph(
+              input_shapes, static_input_map, &graph_for_current_encapsulate,
+              ng_function));
+          int json_indentation = 4;
```

Comment:
Curious, why are we translating here? 
Is is to attach the ng_function in tf2ngraph? The ngfunction is different for different shapes in case the graph has static inputs, right?

Reply:
AOT is only supported for certain kinds of graphs. static inp is not suported. https://github.com/tensorflow/ngraph-bridge/blob/master/ngraph_bridge/ngraph_encapsulate_clusters.cc#L712